### PR TITLE
[feature] FLIT for federated graph classification/regression

### PR DESCRIPTION
**Pull request by wanghh7**

### CODE REVIEW

File: federatedscope/main.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 
-DEV_MODE = False  # simplify the federatedscope re-setup everytime we change the source codes of federatedscope
+DEV_MODE = True  # simplify the federatedscope re-setup everytime we change the source codes of federatedscope
```
Comment (rayrayraykk) [inline federatedscope/main.py:4]: `False`.

File: federatedscope/main.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
     data, modified_cfg = get_data(config=init_cfg.clone())
     init_cfg.merge_from_other_cfg(modified_cfg)
 
-    init_cfg.freeze()
+    # init_cfg.freeze()
```
Comment (rayrayraykk) [inline federatedscope/main.py:35]: Why comment this line?

**Review by rayrayraykk — COMMENTED** [review]
Please see the inline comments, thanks.

**Review by yxdyc — COMMENTED** [review]
It seems that this PR was rebased and committed some existing codes, leading to +83,232 different lines and there are ``Empty file.`` logs for many changed files, e.g., see scripts/format.sh. 

Please consider addressing this before merging, to avoid the strange commit history.

**Review by joneswong — APPROVED** [review]
approved!

**Pull request merged**